frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -73,18 +77,70 @@ export interface PollworkerScreenProps {
   screenReader: ScreenReader;
   printer: Printer;
   togglePollsOpen: () => void;
-  tallyOnCard?: PrecinctScannerCardTally;
-  clearTalliesOnCard: () => Promise<void>;
```

Comment:
The only major refactor I made to BMD was to move the reading/clearing of the precinct tally from the pollworker card into the PollworkerScreen component instead of doing it in AppRoot. I thought this would be a bit clearer since the card is guaranteed to be ready for reading when the pollworker is authenticated.